ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -0,0 +1,304 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Delegate protocol to handle user events inside the side tab bar.
+@objc(MSFSideTabBarDelegate)
+public protocol SideTabBarDelegate {
+    /// Called after the view representing `TabBarItem` is selected.
+    /// - Parameter sideTabBar: The side tab bar.
+    /// - Parameter item: The selected tab bar item.
+    /// - Parameter fromTop: true if the item was in the top section, false if it was in the bottom section.
+    @objc optional func sideTabBar(_ sideTabBar: SideTabBar, didSelect item: TabBarItem, fromTop: Bool)
+
+    /// Called after the avatar view is tapped in the side tab bar.
+    /// - Parameter sideTabBar: The side tab bar.
+    /// - Parameter avatarView: The avatar view.
+    @objc optional func sideTabBar(_ sideTabBar: SideTabBar, didActivate avatarView: AvatarView)
+}
+
+/// View for a vertical side tab bar that can be used for app navigation.
+/// Optimized for horizontal regular + vertical regular size class configuration. Prefer using TabBarView for other size class configurations.
+@objc(MSFSideTabBar)
+open class SideTabBar: UIView {
+    /// Delegate to handle user interactions in the side tab bar.
+    @objc public weak var delegate: SideTabBarDelegate?
+
+    /// The avatar view that displays above the top tab bar items.
+    @objc open var avatarView: AvatarView? {
+        willSet {
+            if let gestureRecognizer = avatarViewGestureRecognizer {
+                avatarView?.removeGestureRecognizer(gestureRecognizer)
+                avatarViewGestureRecognizer = nil
+            }
+
+            avatarView?.removeFromSuperview()
+        }
+        didSet {
+            if let view = avatarView {
+                view.translatesAutoresizingMaskIntoConstraints = false
+                addSubview(view)
+
+                let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleAvatarViewTapped))
+                avatarViewGestureRecognizer = tapGesture;
```

Comment:
the gesture is independent of the new avatartView, we perhaps don't have to create it every time and it can be a let